src/CftfBundle/Resources/views/LsDoc/index.html.twig

```diff
@@ -1,58 +1,102 @@
-            </div>
-            <div class="panel-body js-github-list">
-                <p class="js-github-message-error" style="display: none;">Something went wrong while we are getting github information or you have not signed in with Github, do it <a href="{{ path('github_login') }}">here</a>.</p>
-                <p class="js-github-message-loading">Loading...</p>
-            </div>
-        </div>
+<h1>Competency Frameworks
+    {% if is_granted('create', 'lsdoc') %}
+    <ul class="list-inline pull-right">
+        <li>
+            <a class="btn btn-default" href="{{ path('lsdoc_new') }}">Create a new Framework</a>
+        </li>
+        <li>
+            <a id="js-wizard-btn" class="btn btn-default" href="#">Import Framework</a>
+        </li>
+    </ul>
     {% endif %}
+</h1>
 
-    <h1>Competency Frameworks
-        {% if is_granted('create', 'lsdoc') %}
-            <ul class="list-inline pull-right">
-                <li>
-                    <a class="btn btn-default" href="{{ path('lsdoc_new') }}">Create a new Framework</a>
-                </li>
-            </ul>
-        {% endif %}
-    </h1>
-
-    <div id="competencyTree">
-        {% set lastCreator = '' %}
-        <ul id="comptencyTreeData" style="display:none;">
+<div id="competencyTree">
+    {% set lastCreator = '' %}
+    <ul id="comptencyTreeData" style="display:none;">
         {% for framework in lsDocs %}
-            {% if framework.creator != lastCreator %}
-                {% if not loop.first %}
-                </ul></li>
-                {% endif %}
-                <li class="folder expanded">{{ framework.creator }}<ul>
-                {% set lastCreator = framework.creator %}
+        {% if framework.creator != lastCreator %}
+        {% if not loop.first %}
+    </ul></li>
+    {% endif %}
+    <li class="folder expanded">{{ framework.creator }}<ul>
+            {% set lastCreator = framework.creator %}
             {% endif %}
             <li class="salt-framework-link"><a href="{{ path('editor_lsdoc', { 'id': framework.id }) }}">{{ framework.title }}</a></li>
             {% if loop.last == true %}
-                </ul></li>
-            {% endif %}
+        </ul></li>
+        {% endif %}
         {% endfor %}
         </ul>
+</div>
+
+{% if is_granted('create', 'lsdoc') %}
+<div id="wizard" class="modal fade bs-example-modal-lg" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel">
+    <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4>Import Framework</h4>
+            </div>
+            <div class="modal-body">
+                <div class="row">
+                    <div class="col-md-12">
+                        ASN:
+                        <input name="import" type="radio" value="asn" checked="checked"></input>
+                        Github:
+                        <input name="import" type="radio" value="github"></input>
```

Comment:
This should probably not show up unless github has been configured (github_client_id has a useful value), otherwise is guaranteed to just end up erroring for the end user.

We should probably also look at adding some documentation for an administrator on how to get the github client id and secret and how to configure it (part of a more general question about documentation).